README.md

```diff
@@ -14,149 +19,64 @@ If you want to install a specific version, run the following command with the re
-    - **FNDA:2,calculator**: The `calculator` function was executed 2 times.
-
-4. **Function Summary**
-    - **FNF:3**: The number of functions found in the source file. There are 3 functions: `add`, `multiply`,
-      and `calculator`.
-    - **FNH:2**: The number of functions that were hit. 2 out of the 3 functions were
-      executed: `add` and `calculator`.
-
-5. **Line Execution Details**
-    - **DA:\<line number\>,\<hit count\>**: Indicates whether each line was executed and how many times.
-
-   Here's the details of line coverage:
-
-   | Line | Hits | Explanation                                      |
-   |------|------|--------------------------------------------------|
-   | 7    | 2    | Line 7 (definition of `add`) hit 2 times         |
-   | 8    | 2    | Line 8 (body of `add`) hit 2 times               |
-   | 11   | 0    | Line 11 (definition of `multiply`) not hit       |
-   | 12   | 0    | Line 12 (body of `multiply`) not hit             |
-   | 15   | 2    | Line 15 (definition of `calculator`) hit 2 times |
-   | 16   | 2    | Line 16 (start of `match`) hit 2 times           |
-   | 17   | 2    | Line 17 (call to `add` in `match`) hit 2 times   |
-   | 18   | 0    | Line 18 (call to `multiply` in `match`) not hit  |
-   | 19   | 2    | Line 19 (end of `match`) hit 2 times             |
-
-6. **Line Coverage Summary**
-    - **LF:9**: The total number of lines in the file is 9.
-    - **LH:6**: The total number of lines that were hit (executed at least once) is 6.
-
-7. **End of Record**
-    - **end_of_record**: Indicates the end of this coverage record.
-
-### Summary
-
-1. **Functions Coverage**:
-    - Three functions are defined.
-    - Two functions are executed at least once: `add` and `calculator`.
-    - One function (`multiply`) was not executed.
-
-2. **Line Coverage**:
-    - 9 lines of code in total.
-    - 6 lines were executed during testing.
-    - Lines 11 and 12 (`multiply` function) and line 19 (match arm for `Multiply` operation) were not executed.
-
-> 📝 **Note**
->
-> This format is for a single file. If there are multiple files, each file's report will be concatenated together
-> with `end_of_record` separating them, like this:
-> ```lcov
-> TN:
-> SF:/path/to/your/project/src/operations.cairo
-> FN:7,8,add
-> FN:11,12,multiply
-> FNDA:2,add
-> FNDA:0,multiply
-> ... other metrics ...
-> end_of_record
-> TN:
-> SF:/path/to/your/project/src/lib.cairo
-> FN:3,8,calculator
-> FNDA:2,calculator
-> ... other metrics ...
-> LH:10
-> end_of_record
-> ```
+### Coverage statistics
+
+Various tools exist that can produce coverage statistics based on an lcov file. One of the most well-known among these is [genhtml](https://github.com/linux-test-project/lcov/blob/master/bin/genhtml),
+a tool that generates an html summary report using the coverage data in an lcov file, `genhtml` is part of the [lcov package](https://github.com/linux-test-project/lcov/tree/master).
+
+## Usage in GitHub actions
```

Comment:
This should be much, much earlier in the readme
This is the way most people will use it